### PR TITLE
Reduce boilerplate

### PR DESCRIPTION
**Pull request by johnjohndoe**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -22,9 +22,12 @@ android {
 
     lintOptions {
         warning 'MissingTranslation'
-        warning 'InvalidPackage' // https://github.com/square/okio/issues/58
+        disable 'InvalidPackage' // http://jakewharton.github.io/butterknife/
```
Comment (ligi) [inline app/build.gradle]: ?

Comment (johnjohndoe) [inline app/build.gradle]: That is the recommendation of Butterknife.

Comment (ligi) [inline app/build.gradle]: link? But a) let's not disable this check - let's just not let it break
the build b) I think the url before was more helpful than the new one

On 06.10.2015 14:18, Tobias Preuss wrote:
>  disable 'InvalidPackage'


-- 
best regards,
ligi
http://ligi.de

Comment (johnjohndoe) [inline app/build.gradle]: The link is the Butterknife documentation. I can however revert the change and add the Butterknife docs as second reference.

**Comment by cbalster** [comment]
:+1: for ViewHolder
Regarding butterknife: As much as I like it, I personally wouldn't include it here. IMHO the main advantage in using it is you don't have to type all that boilerplate in the first place as it doesn't bring any functional improvements. Converting it afterwards doesn't make much sense to me. More importantly though I expect this to be an additional hurdle for people potentially wanting to contribute who are not familiar with it.

**Comment by ligi** [comment]
What do we do with this now? I like the change - but have no strong opinion on it - would merge but respect @cbalster 's opinion on it.

**Comment by cbalster** [comment]
I'm still not to sure about his, but we can bring it in if you both like to.
One thing though: If I'm not mistaken we have to call ButterKnife.unbind(this) in the fragment's onDestroyView(). Otherwise we'll keep a reference to the view and leak it.

**Comment by johnjohndoe** [comment]
@cbalster This is a valid point. Sorry for that. I fixed this now.

**Comment by cbalster** [comment]
:+1:

**Comment by johnjohndoe** [comment]
:-)

**Pull request merged**